db/migrations/20230207100201_scents.sql

```diff
@@ -0,0 +1,8 @@
+-- migrate:up
+CREATE TABLE scents (
```

Comment:
테이블 명을 `product_scents`로 수정해주면 좋을 것 같습니다.
```suggestion
CREATE TABLE product_scents (
```